File: python/code/129_pattern21.py
n = int(input("Enter n :- "))
for i in range(n):
    for j in range(i+1):
        if(j==0 or i==n-1 or i == j):
            print("*", end='')
        else:
            print(" ", end='')
    print()
    
# Output :-
# Enter n :- 5
# *
# **
# * *
# *  *
# *****

File: c/code/129_pattern_21.c
#include<stdio.h>
int main() {
    int n;
    printf("Enter n :- ");
    scanf("%d",&n);
    for(int i=0;i<n;i++) {
        for(int j=0;j<=i;j++) {
            if(j==0 || i==n-1 || i == j) printf("*");
            else printf(" ");
        }
        printf("\n");
    }
}

/*

Output :-
Enter n :- 5
*
**
* *
*  *
*****

*/